ci-bench/src/main.rs

```diff
@@ -210,31 +210,31 @@ fn all_benchmarks() -> anyhow::Result<Vec<Benchmark>> {
 static ALL_BENCHMARK_PARAMS: &[BenchmarkParams] = &[
     BenchmarkParams::new(
         KeyType::Rsa,
-        rustls::cipher_suite::TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+        rustls::crypto::ring::cipher_suite::TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
```

Comment:
I think the library changes for this make sense, but IMO all of the example code and tests should probably repeat the path to the suite constants and import them in a `use` statement?